### ISSUE TITLE
Implementing in Core with Javascript/Typescript front-end?

### ISSUE THREAD

**Issue by Blurbs** [alanjuden/MvcReportViewer#76]
I'm wondering if it's possible to implement this package with a Javascript/Typescript front-end as opposed to Razor pages. In my controller if I try to return ViewReportPage(reportPath, page), I get an error saying "Array cannot be null" "Parameter name: bytes". I may be completely misusing this package, so if I am please let me know. Any help would be greatly appreciated!

**Comment by Blurbs** [comment]
Turns out my reports required authentication which I was not passing. This is what the error message was referring to.

**Comment by alicia90** [comment]
How did you solve this? 
Some of my reports have this error, but only those that do not have default parameters.

**Comment by Blurbs** [comment]
@alicia90 In my controller where I'm overriding "NetworkCredentials", I'm returning new NetworkCredential with my domain authentication info. I have my ReportServerUrl overridden as well. Then in my actually report method I'm calling 
`return ViewReportPage("/" + reportName, page);`

I also ended up having to mess with the sql reporting server configuration a good bit to get that working. Changing settings, adding (and then removing) authentication types from the raw config file. 

We ended up ditching sql reporting services for a solution we could bake into our application. Hope any of this helped!
